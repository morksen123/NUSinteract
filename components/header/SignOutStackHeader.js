import * as React from 'react';
import { Appbar } from 'react-native-paper';

const SignOutStackHeader = ({ navigation, back }) => {

  return (
    <Appbar.Header style={{backgroundColor: '#9E89FE'}} dark={false}>
    {back ? <Appbar.BackAction onPress={navigation.goBack} /> : null}
      <Appbar.Content title="Sign Up"/>
    </Appbar.Header>
  );
};

export default SignOutStackHeader;