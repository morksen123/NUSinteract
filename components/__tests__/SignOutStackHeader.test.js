import React from "react";
import renderer from 'react-test-renderer';
import SignOutStackHeader from '../header/SignOutStackHeader';


test('renders correctly', () => {
    const tree = renderer.create(<SignOutStackHeader/>).toJSON();
    expect(tree).toMatchSnapshot();
  });