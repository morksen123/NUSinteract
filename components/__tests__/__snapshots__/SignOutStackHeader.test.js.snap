// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "backgroundColor": "#9E89FE",
        "elevation": 4,
        "zIndex": 0,
      },
      Object {
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 3,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 4,
      },
    ]
  }
>
  <View
    collapsable={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#9E89FE",
        "elevation": 0,
        "flexDirection": "row",
        "height": 56,
        "marginTop": 0,
        "paddingHorizontal": 4,
      }
    }
  >
    <View
      style={
        Object {
          "width": 48,
        }
      }
    />
    <View
      accessibilityState={
        Object {
          "disabled": true,
        }
      }
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "flex": 1,
            "paddingHorizontal": 12,
          },
          Array [
            false,
            Object {
              "alignItems": "center",
            },
            undefined,
          ],
        ]
      }
    >
      <Text
        accessibilityRole="header"
        accessibilityTraits="header"
        accessible={true}
        numberOfLines={1}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Object {
              "textAlign": "left",
            },
            Array [
              Object {
                "color": "#000000",
                "fontFamily": "System",
                "fontWeight": "400",
              },
              Object {
                "fontSize": 17,
              },
              undefined,
            ],
          ]
        }
      >
        Sign Up
      </Text>
    </View>
    <View
      style={
        Object {
          "width": 48,
        }
      }
    />
  </View>
</RCTSafeAreaView>
`;
